pkgs/tools/networking/rpki-arin-tal/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchurl, writeScript, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "rpki-arin-tal";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://www.arin.net/resources/manage/rpki/arin.tal";
+    hash = "sha256-T2weRW/lq0aL6sFJXlfZmh7uqk2fnjRRnq9YhXwhr0g=";
+  };
+
+  builder = writeScript "builder.sh" ''
+    source $stdenv/setup
+    mkdir -p $out
+    cp $src $out/arin.tal
+  '';
+
+  meta = with lib; {
+    description = ''
```

```suggestion
    description = "...";
    longDescription = ''
```
please fill in description